charts/mapproxy/templates/deployment.yaml

```
@@ -71,6 +71,10 @@ spec:
             - name: mapproxy-yaml
               mountPath: /srv/mapproxy/customconfig/
             {{- end }}
+            {{- if and .Values.customSeedConfig.enabled }}
```

I think the `and` isn't really needed
```suggestion
            {{- if .Values.customSeedConfig.enabled }}
```